components/layout_2020/table/layout.rs

```diff
@@ -306,11 +399,11 @@ impl<'a> TableLayout<'a> {
         for column_index in 0..self.table.size.width {
             let column_measure = &mut column_measures[column_index];
             let final_intrinsic_percentage_width = column_measure
-                .percentage_width
+                .percentage
                 .0
                 .min(100. - total_intrinsic_percentage_width);
```

Comment:
Just realized that `Percentage` is actually 1-based, but we can fix them in a follow-up.